packages/fxa-content-server/tests/functional_regression.js

```diff
@@ -48,10 +50,10 @@ module.exports = testsSettingsV2.concat([
   'tests/functional/sign_in_blocked.js',
   'tests/functional/sign_in_cached.js',
   'tests/functional/sign_in_recovery_code.js',
+  'tests/functional/sign_up_with_code.js',
   'tests/functional/sign_up.js',
   'tests/functional/sync_v1.js',
   'tests/functional/sync_v2.js',
   'tests/functional/sync_v3_force_auth.js',
   'tests/functional/tos.js',
-  'tests/functional/verification_reminders.js',
```

Comment:
This test gets rate limited against stage because it has too many failed login attempts from a single IP address. Disabled for now until we up the limit for this.